src/main/java/org/folio/helper/CheckinReceivePiecesHelper.java

```diff
@@ -783,6 +784,30 @@ protected void addErrorForUpdatingItem(Piece piece, Throwable e) {
     }
   }
 
+  protected Map<String, Piece> getProcessedPiecesForPoLine(String poLineId, Map<String, List<Piece>> piecesGroupedByPoLine) {
+    return StreamEx
+      .of(piecesGroupedByPoLine.getOrDefault(poLineId, Collections.emptyList()))
+      .toMap(Piece::getId, piece -> piece);
+  }
+
+  protected Map<String, Integer> getEmptyResultCounts() {
```

Comment:
Maybe (if map is not modified later)
```
    return Map.of(
      ProcessingStatus.Type.SUCCESS.toString(), 0,
      ProcessingStatus.Type.FAILURE.toString(), 0
    );
```

Reply:
The values are updated, that's why didn't use .of()

Reply:
Actually map is modified later. Why we have `Map<String, int>` instead of `Map<ProcessingStatus.Type, int>`?

Reply:
That's a good point. I just refactored some irrelevant code and didn't go much into details. Will modify